.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: false
+language: java
+cache:
+  directories:
+  - $HOME/.m2
+jdk:
+  - openjdk8
+  - openjdk11
+os:
+  - linux
+install:
```

Comment:
I really don't get the travis approach of grabbing dependencies first. Let's use `install: true`, wdyt?